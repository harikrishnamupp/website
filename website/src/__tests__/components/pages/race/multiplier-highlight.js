import React from 'react'
import renderer from 'react-test-renderer'
import MultiplierHighlight from '~components/pages/race/multiplier-highlight'

describe('Components : Race : Multiplier Highlight : nearly 1.5 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={1.4} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : 1.5 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={1.5} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : more than 1.5 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={1.7} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : nearly 2 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={1.9} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : 2 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={2.0} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : more than 2 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={2.2} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : nearly 2.5 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={2.4} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : 2.5 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={2.5} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : more than 2.5 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={2.7} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : nearly 3 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={2.9} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : 3 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={3.0} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : more than 3 times', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={3.2} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Components : Race : Multiplier Highlight : much', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MultiplierHighlight multiplier={4.0} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
