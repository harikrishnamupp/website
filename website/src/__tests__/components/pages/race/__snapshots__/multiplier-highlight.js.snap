// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Race : Multiplier Highlight : 1.5 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      1.5 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : 2 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      2 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : 2.5 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      2.5 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : 3 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      3 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : more than 1.5 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      more than 1.5 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : more than 2 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      more than 2 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : more than 2.5 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      more than 2.5 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : more than 3 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      more than 3 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : much renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      much
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : nearly 1.5 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      nearly 1.5 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : nearly 2 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      nearly 2 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : nearly 2.5 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      nearly 2.5 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;

exports[`Components : Race : Multiplier Highlight : nearly 3 times renders correctly 1`] = `
<div
  className="highlight"
>
  <p>
    This means Black people are dying at a rate
     
    <span
      className="number"
    >
      nearly 3 times
       higher
    </span>
     
    than their population share.
  </p>
</div>
`;
